src/aerovaldb/jsondb/jsonfiledb.py

```diff
@@ -688,7 +688,7 @@ async def query(
         self, asset_type: AssetType | Iterable[AssetType] | None = None, **kwargs
     ) -> list[QueryEntry]:
         if asset_type is None:
-            asset_type = set([AssetType(x) for x in ALL_ROUTES])
+            asset_type = set([AssetType(x) for x in [x.value for x in AssetType]])
```

Comment:
tripple is not always better (and I still don't like the set):
```python
 asset_type = set(AssetType)
```

Reply:
Or even better since you don't need the set but just a `Iterable[AssetType]`:
```python
asset_type = AssetType
```
since an Enum is an Iterable.